### PR TITLE
Edited makefiles to improve testing workflow.

### PR DESCRIPTION
**Pull request by myersCody**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -64,17 +64,15 @@ Testing for RHEL7 or RHEL8
...
     make test-7
     sudo su
+    make setup-yum-7
 
-1b. Additional steps for RHEL8::
+1b. Install Runtime Requirements
 
-    cd /rcm_data/
-    ./setup_rhel8_yum.sh
     yum install -y python36
```
Comment (kholdaway) [inline README.rst]: Can't this just be added to the makefile?  I can't see a reason we document that as a step.

File: rcm_data/Makefile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+DATE		= $(shell date)
+PYTHON		= $(shell which python)
+
+TOPDIR = $(shell pwd)
+
+help:
```
Comment (kholdaway) [inline rcm_data/Makefile]: Why do we need a new makefile?  Can this be moved to the top level Makefile or to the lower level make files?

File: README.rst [hunk 3]
```diff
@@ -64,17 +64,12 @@ Testing for RHEL7 or RHEL8
 
     make test-7
     sudo su
+    cd /rcm_data/deps/certifi/
+    make init-test-7
```
Comment (abaiken) [inline README.rst]: I don't think this is loading correctly if you view the file

Comment (kholdaway) [reply README.rst]: Agree it is not

Comment (myersCody) [reply README.rst]: fixed

**Pull request merged**